CONTRIBUTING.md

```diff
@@ -136,7 +135,18 @@ So the step would be:
 The environment variable `GITHUB_TOKEN` is needed to publish at GitHub and it must
 point to a token generated by github to access projects.
 
-# CI infraestructure
+# Profiling
+
+There is a possibility to enable golang pprof profiler.
+ - Enable profiler in `operator.yaml` by changing value of 'ENABLE_PROFILER' to True
+ - You can change profiler port by editing 'PROFILER_PORT' - default is 6060
+ - Deploy new code to cluster - example:  "make cluster-sync"
+ - Find nmstate-handler pod name - "kubectl get pods -n nmstate"
+ - Create port forwarding to pod - example: "kubectl port-forward pod/<pod name> 6060:6060 -n nmstate"
```

Comment:
this renders as "pod/", <> are omitted.

Would you please keep all command examples as inline code (enclose them within `)